### PR TITLE
Add keyboard support for confirmation delete button [Editor-design-view]

### PR DESCRIPTION
**Pull request by LahiruRajapaksha**
## Preview
image1

### CODE REVIEW

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js [hunk 1]
```diff
@@ -1466,6 +1503,10 @@ define(['require', 'log', 'lodash', 'jquery', 'partition', 'stream', 'query', 'f
             });
 
             newElement.on('keydown', function (key) {
+                if ( key.keyCode == 46 || key.keyCode == 8 ) {
```
Comment (AnuGayan) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js]: shall we introduce a constant for the 46, 8, 13, 9, 39, 37 and 27?

**Comment by Niveathika92** [comment]
@LahiruRajapaksha Shall we revisit the stylings of the button? IMO the selected button having white letters is bit confusing.

**Comment by LahiruRajapaksha** [comment]
new

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js (hunk 1 above)
Comment (LahiruRajapaksha) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js]: Assign constants to keyCodes

**Comment by LahiruRajapaksha** [comment]
new

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js [hunk 2]
```diff
@@ -1014,6 +1016,41 @@ define(['require', 'log', 'jquery', 'backbone', 'lodash', 'designViewUtils', 'dr
...
+                                $('.yes').focus();
+                            }
+                        });
+                        $(".yes").on('keydown', function (e) {
+                            if (e.keyCode == 37) {
+                                $('.no').focus();
+                            }
+                        });
+                        $(".no").on('keydown', function (e) {
+                            if (e.keyCode == 9) {
```
Comment (AnuGayan) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js]: Shall we replace these as well with a constant

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js [hunk 3]
```diff
@@ -978,14 +983,16 @@ define(['require', 'log', 'jquery', 'backbone', 'lodash', 'designViewUtils', 'dr
...
+                            title: 'Confirmation',
                             html: true,
                             content: function () {
                                 return $('.pop-over').html();
 
                             }
                         });
+                        $('#' + connectionObject.id).off();
                         $('#' + connectionObject.id).popover("show");
+                        $('.no').focus();
```
Comment (dnwick) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js]: please use a better class name than "no"

Comment (LahiruRajapaksha) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js]: Added the class names for yes and no buttons as 'btn_yes' and 'btn_no' respectively

**Pull request merged**